src/Pages/ChannelRequests.razor

```diff
@@ -46,9 +42,18 @@
         <PopupTitleTemplate>
             <h2>Create Channel Request</h2>
         </PopupTitleTemplate>
+
         <ChildContent>
+            <DataGridColumn>
+                <CaptionTemplate>
+                      @if (_isNodeManager) {
+                    <Button Color="Color.Success" TextColor="TextColor.Light" Clicked="OnShowNewChannelRequestModal">New</Button>
```

Comment:
I am not sure if this is what we want. AFAIK we are using [this](https://blazorise.com/docs/extensions/datagrid/templates/commands) for the rest of the pages. 😄

Reply:
Fixed the issue.